apps/site/lib/site_web/controllers/fare_controller.ex

```diff
@@ -99,7 +99,12 @@ defmodule SiteWeb.FareController do
               Enum.join(summary.name)
             end
 
-          {String.capitalize(name), Fares.Summary.price_range(summary)}
+          # capitalize lowercases every word after the first word in `name`.  This fixes the one
+          # edge case for Commuter Rail
+          capitalizedName = String.capitalize(name)
+          changedName = String.replace(capitalizedName, "Commuter rail", "Commuter Rail")
```

Comment:
nitpick(not blocking): this and the if/else above could probably be extracted together into a `format_name` function for better readability.